todo_app/app.py

```diff
@@ -3,21 +3,87 @@
 from todo_app.flask_config import Config
 
 import todo_app.data.session_items as session_items
+import requests, os, json
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+class Item:
```

Comment:
It would be nice to break this file up a bit - this Item class could easily have its own file, and similarly the new functions to interact with Trello could make up a `trello_service.py` file or similar

Reply:
Broke it down. We have another module now that takes care of Item class and functions, and only necessary functions have been imported to app.py